hermezconfig-cardona-internal.yaml.example

```diff
@@ -1,4 +1,4 @@
-datadir: /path/to/datadirs/hermez-cardona-internal
+datadir: ~/.cdk-erigon/hermez-cardona-internal
```

Comment:
What about to use this for all?
```
~/.cdk-erigon/data
```
with this it would not be necessary to specify the datadir in docker